app/controllers/items_controller.rb

```diff
@@ -4,14 +4,14 @@ class ItemsController < ApplicationController
   def index
     @items = current_organization
       .items
-      .includes(:base_item, :kit, :line_items, :request_units)
+      .includes(:base_item, :kit, :line_items, :request_units, :item_category)
       .alphabetized
       .class_filter(filter_params)
       .group('items.id')
     @items = @items.active unless params[:include_inactive_items]
 
     @item_categories = current_organization.item_categories.includes(:items).order('name ASC')
-    @kits = current_organization.kits.includes(line_items: :item, inventory_items: :storage_location)
+    @kits = current_organization.kits.active.includes(line_items: :item, inventory_items: :storage_location)
```

Comment:
Don't think you can do it here because then in the kits tab the activate/deactivate button pulled in from app/views/kits/_table.html.erb won't work